### ISSUE TITLE
Can't build with nix on OS X due to missing glibcLocales

### ISSUE THREAD

**Issue by jjant** [radicle-dev/radicle#639]
Getting the following error
```nix-shell nix/shell.nix --show-trace
error: while evaluating the attribute 'shellHook' of the derivation 'radicle-dev' at /nix/store/dmj99vvab9afi6za2a5b1sjg8j47ij2n-source/pkgs/stdenv/generic/make-derivation.nix:177:11:
cannot coerce null to a string, at /Users/jant/work/radicle/nix/shell.nix:25:17
```

It appears to be that glibcLocales is not available in OS X.

Removing the `LOCALE_ARCHIVE` line which contains the call to `glibcLocales` seems to work (for now at least)

**Comment by jkarni** [comment]
Thanks for the report!

I don't have a Mac with nix to test with, but could you try with #640?

**Comment by jkarni** [comment]
Should be fixed with #646 - please reopen if that's not the case.
